src/browser.rs

```diff
@@ -559,11 +559,18 @@ async fn ws_url_from_output(
     }
 }
 
+#[derive(Debug, Clone)]
+enum HeadlessMode {
+    False,
+    True,
+    New,
+}
```

Comment:
can we document these?

Reply:
Hi @mattsse sure and it's now added. Sorry for not having included any description from the start.